### DISCUSSION TITLE
Dockerisation of the project

### DISCUSSION THREAD

**Discussion by hvmzx** [K-vanc/Tempest-EPG-Generator#9] [Ideas]
It would be nice if Tempest was containerised so it can be run in a container

**Comment by K-vanc** [comment]
Hi and thanks for using Tempest.

Creating a docker build is one of the item in my road map since I received some requests from users on OSes such as CoreElec which does not have native php support. I conceptionally thinking a container such as "Nginx/Apache + PHP + Tempest" so it may run on any enviroment which Docker supported by just user mounted siteconfig folders but the main problem is I never used/configured docker before and currently I have no free time to focus on it due to  heavy traffic in my real life job.. I read some articles about how to create and set docker but not yet tried to make a build

**Reply by Jimpan81** [reply]
Any News absolut docker ?

**Reply by K-vanc** [reply]
Hi. Unfortunately couldn't have enough free time yet to learn how to set, use and publish a docker container.. 

In order not to keep people waiting more, I am open to any help hand from devs/coders with docker experience.

**Comment by K-vanc** [comment]
Hi guys;

Finally, some progress and update for docker;
https://github.com/K-vanc/Tempest-EPG-Generator/issues/17#issuecomment-1762748587

**Comment by K-vanc** [comment]
And docker image released for all available architectures on alpine;

https://hub.docker.com/r/kvanc/tempest_epg
